<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PottsGUI</class>
 <widget class="QWidget" name="PottsGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Potts: Please define CPM configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_19">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_general">
        <attribute name="title">
         <string>General</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>Lattice Dimensions</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_25">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_24">
              <item>
               <widget class="QLabel" name="label_28">
                <property name="text">
                 <string>x</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="xDimSB">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>10000000</number>
                </property>
                <property name="value">
                 <number>256</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_29">
                <property name="text">
                 <string>y</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="yDimSB">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>10000000</number>
                </property>
                <property name="value">
                 <number>256</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_30">
                <property name="text">
                 <string>z</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="zDimSB">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>10000000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Boundary Conditions (Cell Lattice)</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>x</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="xbcCB">
                <item>
                 <property name="text">
                  <string>NoFlux</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Periodic</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>y</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="ybcCB">
                <item>
                 <property name="text">
                  <string>NoFlux</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Periodic</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>z</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="zbcCB">
                <item>
                 <property name="text">
                  <string>NoFlux</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Periodic</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="5" column="0">
            <widget class="QLabel" name="label_33">
             <property name="text">
              <string>Number of MC Steps</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Fluctuation Function</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QSpinBox" name="mcsSB">
             <property name="maximum">
              <number>1000000000</number>
             </property>
             <property name="value">
              <number>100000</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_44">
             <property name="text">
              <string>LatticeType</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="latticeTypeCB">
             <item>
              <property name="text">
               <string>Square</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Hexagonal</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_32">
             <property name="text">
              <string>Pixel Copy Range (NeighborOrder)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="fluctuation_FcnCB"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_31">
             <property name="text">
              <string>Average Membrane Fluctuations</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="membraneFluctuationsLE">
             <property name="toolTip">
              <string>Also known as so called temperature parameter in the acceptance probability expresion:  exp(-delta E/(k*T))</string>
             </property>
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="neighborOrderSB">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QComboBox" name="cell_TypeCB"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="type_ParamLE"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_advanced">
        <attribute name="title">
         <string>Advanced</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Boltzmann Acceptance</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Offset</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="offset_SB">
                  <property name="minimum">
                   <double>-99990000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>99990000.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Coefficient</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="coefficient_SB">
                  <property name="maximum">
                   <double>99990000.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Simulation</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <layout class="QFormLayout" name="formLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Anneal Steps</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="anneal_SB">
                  <property name="maximum">
                   <number>99999999</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Flips per Site</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="flip_to_dim_SB">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>99999999</number>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QSpinBox" name="debug_SB">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>99999999</number>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QSpinBox" name="random_seed_SB">
                  <property name="minimum">
                   <number>0</number>
                  </property>
                  <property name="maximum">
                   <number>99999999</number>
                  </property>
                  <property name="value">
                   <number>0</number>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QCheckBox" name="random_seed_CB">
                  <property name="text">
                   <string>Random Seed</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QCheckBox" name="debug_CB">
                  <property name="text">
                   <string>Debug Output Frequency</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Energy Function Calculator</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,0,0,0,0,0,0,0" columnminimumwidth="0,0,0,0,0,0,0,0">
                <item row="1" column="0">
                 <widget class="QCheckBox" name="ef_results_CB">
                  <property name="text">
                   <string>Results</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QCheckBox" name="ef_enable_CB">
                  <property name="text">
                   <string>Enable</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QCheckBox" name="ef_accepted_CB">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Accepted</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="ef_rejected_CB">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Rejected</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="1" column="2">
                 <widget class="QCheckBox" name="ef_total_CB">
                  <property name="text">
                   <string>Total</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Spin Flip Frequency</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QSpinBox" name="ef_spin_freq_SB">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>99999999</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="3" column="0">
               <layout class="QFormLayout" name="formLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>File Name</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="ef_file_name_LE">
                  <property name="text">
                   <string>statData.txt</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Output Frequency</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="ef_file_freq_SB">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>99999999</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
